src/zjs_console.c

```diff
@@ -297,13 +299,22 @@ void zjs_console_init(void)
     jerry_value_t console = jerry_create_object();
 
     zjs_obj_add_function(global_obj, get_console_obj, "Console");
-    zjs_obj_add_function(console, console_log, "log");
-    zjs_obj_add_function(console, console_log, "info");
-    zjs_obj_add_function(console, console_error, "error");
-    zjs_obj_add_function(console, console_error, "warn");
-    zjs_obj_add_function(console, console_time, "time");
-    zjs_obj_add_function(console, console_time_end, "timeEnd");
-    zjs_obj_add_function(console, console_assert, "assert");
+
+    zjs_native_func_t array[] = {
+        { console_log, "log" },
+        { console_log, "info" },
+        { console_error, "error" },
+        { console_error, "warn" },
+        { console_time, "time" },
+        { console_time_end, "timeEnd" },
+        { console_assert, "assert" },
+        { NULL, NULL }
+    };
+
+    console_prototype_obj = jerry_create_object();
+    zjs_obj_add_functions(console_prototype_obj, array);
+
+    jerry_set_prototype(console, console_prototype_obj);
```

Comment:
Good point, I was going off of probably very new Node.js documentation which had the ability to create a new Console object (`new Console()`) but it seems unnecessary and not really needed.